eth-pm/package.json

```diff
@@ -49,7 +48,9 @@
     ]
   },
   "devDependencies": {
+    "@craco/craco": "^6.4.5",
     "@ethersproject/shims": "^5.5.0",
+    "@types/craco__craco": "^6.4.0",
```

Comment:
Where do you use the typing?

Reply:
I was trying to initiialise craco.config.js in ts instead so installed types, but ended up reverting to js because of some weird bugs and, from what it looked like, the lack of TS support.

Pushed a commit to remove this as a dependency.